pkg/httputil/client.go

```diff
@@ -0,0 +1,13 @@
+package httputil
+
+import "net/http"
+
+// GetRequestLakeFSClient get lakeFS client identifier from request.
+//   It extracts the data from X-Lakefs-Client header and fallback to the user-agent
+func GetRequestLakeFSClient(r *http.Request) string {
+	id := r.Header.Get("X-Lakefs-Client")
+	if id == "" {
+		id = r.UserAgent()
```

Comment:
Won't it spam the DB? The cardinality of user-agent is huge

Reply:
I consider 'event' information as raw data - we can filter out the data. Using the user-agent as the client identify will help us also process previous/older clients using the lakefs.